scripts/main.js

```diff
@@ -1 +1,117 @@
-// TODO: add JS
\ No newline at end of file
+// ------* selecting elements that will change with scene selection *--------
+const html = document.documentElement;
+const header = document.querySelector('h1');
+const select = document.querySelector('select');
+const audio = document.querySelector('audio');
+
+// ------* function declarations *---------
+
+function onSceneOptionSelected(event) {
+    const sceneSelection = event.target.value;
+    localStorage.setItem('selection', sceneSelection);
+    const selection = localStorage.getItem('selection');
+
+    const chooseSceneOption = document.querySelector('#choose');
+    if (chooseSceneOption) {
+        chooseSceneOption.remove();
+    }
+    
+    renderSelectedScene(selection);
+}
+
+function onClickToStartMedicationClicked() {
+    const overlay = document.querySelector('#overlay');
+    overlay.remove();
+}
+
+function renderSelectedScene(selection) {
+    renderBackground(selection);
+    renderHeader(selection)
+    renderSelectionBackground(selection);
+    renderAudio(selection);
+}
+
+function renderBackground(selection) {
+    if (selection === 'beach') {
+        html.style.background = 'url("../images/beach.jpeg")';
+        html.style.backgroundSize = 'cover';
+    } else if (selection === 'forest') {
+        html.style.background = 'url("../images/forest.jpeg")';
+        html.style.backgroundSize = 'cover';
+    } else if (selection === 'stream') {
+        html.style.background = 'url("../images/stream.webp")';
+        html.style.backgroundSize = 'cover';
+    } else if (selection === 'rain') {
+        html.style.background = 'url("../images/rain.jpeg")';
+        html.style.backgroundSize = 'cover';
+    }
```

Comment:
You should also have an else statement for if an unexpected string is passed and throw some sort of error